feed/utils.js

```diff
@@ -0,0 +1,50 @@
+async function getActivityFeedData(query = {}, nextLink) {
+  let finalUrl =
+    API_BASE_URL + (nextLink || '/logs' + generateActivityFeedParams(query));
+  const res = await fetch(finalUrl, {
+    credentials: 'include',
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
+  return await res.json();
+}
+
+function generateActivityFeedParams(query) {
```

Comment:
There's a semicolon at the end of the first line, after the variable declaration (let queryParam = 'dev=true&format=feed';). While technically valid, it's considered a better practice to omit semicolons at the end of lines in JavaScript's automatic semicolon insertion (ASI) environment. This can help prevent potential syntax errors or unexpected behavior in certain cases.

Reply:
The semicolon is getting added by prettier, when I ran yarn fix.